mern-ecommerce/controllers/category.js

```diff
@@ -68,3 +68,36 @@ exports.list = (req, res) => {
     res.json(data);
   });
 };
+
+exports.addProductToCategory = (req, res) => {
+  const { categoryId } = req.params;
+  const { productId } = req.body;
+
+  Category.findById(categoryId, (err, category) => {
+    if (err || !category) {
+      return res.status(400).json({
+        error: "Category not found",
+      });
+    }
+
+    Product.findById(productId, (err, product) => {
+      if (err || !product) {
+        return res.status(400).json({
+          error: "Product not found",
```

Comment:
The error message "Product not found" is not consistent with the error message "Category doesn't exist" used in the `categoryById` function. It would be better to use a consistent error message format throughout the code. Consider using a centralized error message handler or a constants file to define error messages.